### PR TITLE
Support section at contribute page

### PR DESCRIPTION
**Pull request by mahmodHammad**
Fixes #

#### Describe the changes you have made in this PR -
- adding lite border & padding to each card 
- fix alignment at md screens 
- fix margin y of support-pay buttons at phone screens 
### Screenshots of the changes 

image

### AT md screens : 
image

###  support-pay buttons at phone screens 
image

### CODE REVIEW

File: app/assets/stylesheets/about.scss [hunk 1]
```diff
@@ -40,17 +40,22 @@
   transform: translateY(-7px);
 }
 
+.contact-container{
```
Comment (tachyons) [inline app/assets/stylesheets/about.scss]: Space before {

**Comment by mahmodHammad** [comment]
@tachyons   @nitin10s  
i got stuck on these 6 code Climate issues plwase help !

File: public/js/listeners.js [hunk 2]
```diff
@@ -1,425 +0,0 @@
...
-    document.getElementById("simulationArea").addEventListener('mousedown', function(e) {
-
-        $("input").blur();
-
-        errorDetected = false;
-        updateSimulation = true;
-        updatePosition = true;
-        updateCanvas = true;
-
-        simulationArea.lastSelected = undefined;
```
Comment (tachyons) [inline public/js/listeners.js]: Why?

Comment (mahmodHammad) [reply public/js/listeners.js]: @tachyons 
i added this files by mistake  , the i deleted it from my PR
- the only changes are at styling 
no thing at .js files are included so i deleted it after  realizing i added them

Comment (mahmodHammad) [reply public/js/listeners.js]: @tachyons  
Please Help me if you know a way to remove some commits from the PR

Comment (mahmodHammad) [reply public/js/listeners.js]: i will google it tomorrow and fix it :)

**Comment by satu0king** [comment]
@mahmodHammad easiest way is to close this PR and create a new one

**Comment by mahmodHammad** [comment]
https://github.com/CircuitVerse/CircuitVerse/pull/1129#issuecomment-602160866 
@nitin10s  @tachyons 
YUP , it seems i messed up the branch while trying to fix it 🤦‍♀️, so i will close it now .
- i will make a cleaner PR with these changes as soon as possible

**Pull request closed without merging**